### ISSUE TITLE
GMT vs Local time

### ISSUE THREAD

**Issue by gobbluth** [giuse88/duka#18]
![screen shot 2016-10-07 at 3 20 01 pm](https://cloud.githubusercontent.com/assets/3258237/19191199/a00b5b74-8ca1-11e6-9e04-4c917287fa39.png)

This should be an option for the CLI tool perhaps since it's possible in the web GUI?

**Comment by giuse88** [comment]
true. I am using UTC time in duka, I can look into implementing it ;)

**Comment by kavuraki** [comment]
Many thanks for the tool.
Any possible progress on this?

**Comment by giuse88** [comment]
I haven't started working on it yet. I am planning to fix these issues in a week ;) I've been quite busy recently ;)

**Comment by giuse88** [comment]
@gobbluth @kavuraki What do you think if I add a new command line option like : 
```
--local-time          use local time (default GMT)
```

**Comment by giuse88** [comment]
Check out Pull request : #19

**Comment by kavuraki** [comment]
@giuse88 
So in this case can I enter the local timezone manually?

**Comment by giuse88** [comment]
@kavuraki 
No, if you select local, it converts the timestamp to your local timezone.

**Comment by kavuraki** [comment]
@giuse88 that's great. It works for me!

**Comment by giuse88** [comment]
@kavuraki Cool! Do you think you can try the code in this branch? It works for me but I live in the GMT timezone. Thus, there is no difference between GMT and local. Testing timezone is quite annoying, if it works for you, I will release it today.

**Comment by kavuraki** [comment]
@giuse88 what should I set for the variable? nothing? is it "True"?

duka EURUSD -s 2016-10-15 -e 2016-11-28 -c D1 --local-time

**Comment by giuse88** [comment]
That should just work

**Comment by kavuraki** [comment]
@giuse88 D1 data still start from 03:00 since I live in UTC+3

Maybe it's not the place btw I get csv files one empty row between each rows.

**Comment by giuse88** [comment]
Can you upload the file here?
You're getting the same data you would get without local-time. The only difference is that now the data timestamp is converted to your local time.

**Comment by kavuraki** [comment]
In this case I got confused. Cause I notice that some days are twice in the table.

`duka EURUSD -s 2016-10-15 -e 2016-11-28 -c D1`

`duka EURUSD -s 2016-10-15 -e 2016-11-28 -c D1 --local-time`

Please find attached output files for the commands above.

[outputs.zip](https://github.com/giuse88/duka/files/619597/outputs.zip)

From Dukascopy If I select "local" the day starts from 0:00 (it's 21:00 in UTC which is 0:00 in UTC+3)

**Comment by giuse88** [comment]
@kavuraki Thanks for your help. ;) I am looking into it..

**Comment by giuse88** [comment]
Yes, I identify the problem you are having. It's caused by daylight time change :( ... It is not related to the local change I made. However, I am not satisfied with the local change. A proper change requires a big refactor and I have no time right now, so I am going to revert this change and merge the rest.

**Comment by taewookim** [comment]
Any updates on this?
